richtext-commonmark/src/commonMain/kotlin/com/halilibo/richtext/markdown/Markdown.kt

```diff
@@ -180,56 +163,31 @@ internal fun RichTextScope.RecursiveRenderMarkdownAst(astNode: AstNode?) {
     // In any case, we should include it here to prevent any
     // non-rendered text problems.
     is AstText -> {
-      Log.e("MarkdownRichText", "Unexpected raw text while traversing the Abstract Syntax Tree.")
-      Text(astNodeType.literal)
+      // TODO(halilozercan) use multiplatform compatible logging
+      Napier.d("Unexpected raw text while traversing the Abstract Syntax Tree.", tag = "MarkdownRichText")
```

Comment:
Is printing to Stderr not sufficient? If someone's looking for a text formatting library, it's unfortunate if they have to also bring in a random third-party logging library.

Reply:
You are definitely right! I was experimenting with something here and forgot to remove it for the final PR :/